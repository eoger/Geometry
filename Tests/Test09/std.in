C Cercle 5 5 5
R Rect 1 2 3 4
UNDO
UNDO
UNDO
LIST
REDO
LIST
REDO
LIST
L Ligne 1 2 3 4
PL PolyL 1 2 3 4 5 6
OA oa1 Cercle Ligne PolyL
OA oa2 oa1 PolyL Rect
OA oa3 oa1 oa2
UNDO
UNDO
LIST
UNDO
LIST
REDO
REDO
REDO
UNDO
L Ligne2 1 2 3 4
LIST
REDO
L Ligne3 1 2 3 4
L Ligne3 1 2 3 4
UNDO
LIST
REDO
LIST
EXIT