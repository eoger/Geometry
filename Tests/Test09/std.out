OK
OK
OK
OK
ERR
OK
C Cercle 5 5 5
OK
C Cercle 5 5 5
R Rect 1 2 3 4
OK
OK
OK
OK
OK
OK
OK
C Cercle 5 5 5
L Ligne 1 2 3 4
OA oa1 Cercle Ligne PolyL
PL PolyL 1 2 3 4 5 6
R Rect 1 2 3 4
OK
C Cercle 5 5 5
L Ligne 1 2 3 4
PL PolyL 1 2 3 4 5 6
R Rect 1 2 3 4
OK
OK
OK
OK
OK
C Cercle 5 5 5
L Ligne 1 2 3 4
L Ligne2 1 2 3 4
OA oa1 Cercle Ligne PolyL
OA oa2 PolyL Rect oa1
PL PolyL 1 2 3 4 5 6
R Rect 1 2 3 4
ERR
OK
ERR
OK
C Cercle 5 5 5
L Ligne 1 2 3 4
L Ligne2 1 2 3 4
OA oa1 Cercle Ligne PolyL
OA oa2 PolyL Rect oa1
PL PolyL 1 2 3 4 5 6
R Rect 1 2 3 4
OK
C Cercle 5 5 5
L Ligne 1 2 3 4
L Ligne2 1 2 3 4
L Ligne3 1 2 3 4
OA oa1 Cercle Ligne PolyL
OA oa2 PolyL Rect oa1
PL PolyL 1 2 3 4 5 6
R Rect 1 2 3 4
